scripts/netContain/Dockerfile

```diff
@@ -24,7 +24,10 @@ LABEL maintainer "Cisco Contiv (https://contiv.github.io)"
 
 RUN mkdir -p /etc/openvswitch \
  && echo 'http://dl-cdn.alpinelinux.org/alpine/v3.4/main' >> /etc/apk/repositories \
- && apk --no-cache add openvswitch=2.5.0-r0 iptables ca-certificates openssl curl bash
```

Comment:
I think a better fix will be to mount /run/contiv from installer ?

Reply:
@rchirakk: I've just reverted it to the way it was before the change to fix the issue with compressed kernel modules. We can fix it in various ways, but it'd be nice not having to update this in all the places (contiv/install, kubespray PR and perhaps other places).

This is a very simple change which lets us maintain backwards compatibility without new requirements.